tests/mmm/components/test_adstock.py

```diff
@@ -0,0 +1,13 @@
+#   Copyright 2024 The PyMC Labs Developers
```

Comment:
Is this file empty?

Reply:
Yes. I will add some tests today

Reply:
Added tests in PR here: https://github.com/cetagostini-wise/pymc-marketing/pull/5

Reply:
merge!